manageiq-operator/pkg/helpers/miq-components/cr.go

```diff
@@ -298,26 +354,38 @@ func ManageCR(cr *miqv1alpha1.ManageIQ, c *client.Client) (*miqv1alpha1.ManageIQ
 		cr.Spec.HttpdAuthenticationType = httpdAuthenticationType(cr)
 		cr.Spec.HttpdImageNamespace = httpdImageNamespace(cr)
 		cr.Spec.HttpdImageTag = httpdImageTag(cr)
+
+		privileged, err := PrivilegedHttpd(cr.Spec.HttpdAuthenticationType)
+		if err != nil {
+			return err
+		}
+
+		cr.Spec.HttpdImage = httpdImage(cr, privileged)
 		cr.Spec.ImagePullSecret = imagePullSecretName(cr, *c)
 		cr.Spec.KafkaImageName = kafkaImageName(cr)
 		cr.Spec.KafkaImageTag = kafkaImageTag(cr)
+		cr.Spec.KafkaImage = kafkaImage(cr)
```

Comment:
From a consistency perspective, in the other places, KafkaImage is sorted before KafkaImageName, so we should do the same here (unless there's an actual code reason).

Same goes for the rest.

Reply:
The reason I put them after the ImageName/ImageTag is that we want to default/define those first since we use them to build the *Image ones if not specified.

Reply:
Ah ok...makes sense.